src/main/java/Controller/PickupSystem.java

```diff
@@ -6,31 +6,70 @@
 public class PickupSystem {
     private UserManager uman = new UserManager();
     private ItemManager iman = new ItemManager();
-    private String currentUser = "";
+    private static String currentUser = "";
 
-    public static boolean userLogin(String username, String pw){
+    public PickupSystem{
+
+    }
+
+    public boolean userLogin(String username, String pw){
         // login
+        if(uman.lookupUser(username,pw) == false){
+            return false;
+        }
         return true;
     }
 
-    public static boolean userRegister(String username, String pw){
+    public boolean userRegister(String username, String pw){
         // lookup username; if already exists, return false.
         // else call Usermanager.register
-        return true;
+        if(uman.lookupUser(username,pw) == false){
+            uman.register();
+            return true;
+        }
+        return false;
     }
 
     public boolean userLogout(){
         // set current user as ""
         return true;
     }
 
-    public void pickup(){
+    public boolean pickup(String id){
         // this will interact with the UI layer
-        iman.removeItem("12345");
+        if(iman.searchItem(id) == null){
+            return false;
+        }
+        else{
+            iman.removeItem(id);
+        return true;
+        }
     }
 
-    public void storeItem(){
+    public boolean storeItem(String id, List<String> info, String storageRequirement){
         // this will interact with the UI layer
+        iman.createItem(String id, List<String> info, String storageRequirement);
+        if(iman.addItem(id,currentUser)==false){
+            return false;
+        }
+        return true;
+
+    public boolean search() {
```

Comment:
Fulfill this method and add a remove method

Reply:
ok got it